PlateSpace/ETL/README.md

```diff
@@ -4,19 +4,24 @@ The purpose of this script is to load restaurant data into a MongoDB Atlas datab
-    - `YELP_AUTH_TOKEN`: here you need to put your OAuth2 bearer token generated from [https://api.yelp.com/oauth2/token](https://api.yelp.com/oauth2/token). More information is available in the [Yelp Developer site](https://www.yelp.com/developers/documentation/v3/authentication).
-  1. Install yarn using this link: https://yarnpkg.com/lang/en/docs/install/
-  1. Install the following packages using yarn: `yarn`
-  1. Run command `node index.js`
-
-The script creates new collection for restaurants, using Yelp Fusion API.
-
-Collection schema:
+1. Create a MongoDB Atlas cluster
+1. [Sign up on Yelp](https://www.yelp.com/signup) to get access to [Yelp's developer site](https://www.yelp.com/developers) and generate a Yelp OAuth authorization token.
+1. Edit the [createIndexes.sh](./createIndexes.sh) file and and set the values the following parameters:
+    * `PRIMARY="Primary Server:27017"`: enter the FQDN (fully qualified domain name) of your MongoDB Atlas primary node (such as `platespace-shard-00-00-abcde.mongodb.net:27017`)
+    * `USERNAME`: enter your Atlas admin username (from the __MongoDB Users__ tab of your MongoDB Atlas cluster)
+    * `PASSWORD`: enter your Atlas admin password
+1. Edit the [config.js](./config.js) file and set the values for the following parameters:
+    * `MONGODB_ATLAS_URI`: Your MongoDB Atlas connection string. Make sure the `<DATABASE>` placeholder is set to `platespace`.
+    * `YELP_AUTH_TOKEN`: add the OAuth2 bearer token generated from [https://api.yelp.com/oauth2/token](https://api.yelp.com/oauth2/token) using the Client ID and Client Secret values from your [Yelp App](https://www.yelp.com/developers/v3/manage_app). More information is available in the [Yelp Developer site](https://www.yelp.com/developers/documentation/v3/authentication). The screenshot below shows how to generate such a token using [Postman](https://www.getpostman.com) and your Yelp app's Client ID and Secret.
+
+    ![Generating a Yelp OAuth token with Postman](https://github.com/mongodb/stitch-examples/blob/master/PlateSpace/ETL/res/YelpOAuthToken_Postman.jpg?raw=true)
+    * `BASE_LOCATION_LATITUDE`: Enter the latitude of the base location that Yelp uses to retrieve the nearest 1,000 restaurants. Use negative decimals for South latitudes. Currently set to the latitude of San Francisco, California.
+    * `BASE_LOCATION_LONGITUDE`: Enter the longitude of the base location that Yelp uses to retrieve the nearest 1,000 restaurants. Use negative decimals for West longitudes. Currently set to the longitud of San Francisco, California.
+1. From a Terminal window running from the __ETL__ folder, run `npm install` to install the required Node packages.
+1. Run `node index.js` to import restaurant data from Yelp into the `restaurants` collection of your `platespace` database.
+1. Run `sh createIndexes.sh` to create the necessary geospatial and unique compound indexes in the `restaurants` and `reviewsRatings` collection.
```

Comment:
I think this is unnecessary now, because `index.js` will also create the index too, right?